planter/planter.sh

```diff
@@ -57,7 +57,7 @@ VOLUMES="-v ${REPO}:${REPO} -v ${HOME}:${HOME} --mount type=tmpfs,destination=${
 GID="$(id -g ${USER})"
 ENV="-e USER=${USER} -e GID=${GID} -e UID=${UID} -e HOME=${HOME}"
 # construct the final docker command
-CMD="docker pull ${IMAGE} && docker run --rm ${VOLUMES} --user ${UID} -w ${PWD} ${ENV} ${DOCKER_EXTRA:-} ${IMAGE} ${@}"
```

Comment:
could you add a comment explaining why we added these or perhaps linking to the issue?
I'd like to try to keep all of planter.sh unsurprising or self-explanatory.